content/docs/gke/troubleshooting-gke.md

```diff
@@ -19,10 +19,18 @@ For more help, try the
 
 Here are some tips for troubleshooting Cloud IAP.
 
- * Make sure you are using HTTPS.
- * See the guide to 
+* Make sure you are GCP project owner
+* Make sure you are using HTTPS.
+* Check project [quota page](https://pantheon.corp.google.com/iam-admin/quotas) to see if there are any quota limit, increase them as needed.
+* Check [deployment manager page](https://console.cloud.google.com/deployments) and see if there’s a failed deployment.
+* Check if endpoint is up: do [DNS lookup](https://mxtoolbox.com/DNSLookup.aspx) against your IAP url and see if can resolve to correct ip.
+* Check if certificate succeeded: “kubectl describe certificates -n kubeflow” should give you certificate status.
+* Make sure you added https://<deployment>.endpoints.<project>.cloud.goog/_gcp_gatekeeper/authenticate 
```

Comment:
I think you need to put the URL in code formatting, otherwise it will be rendered on the webpage.